src/main/java/nextstep/ladder/Position.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.ladder;
+
+public class Position {
+    int position;
```

Comment:
```suggestion
    private int position;
```
인스턴스 변수는 가능하면 private으로 구현